scene/gui/graph_edit.cpp

```diff
@@ -775,6 +775,11 @@ void GraphEdit::_gui_input(const Ref<InputEvent> &p_ev) {
 	}
 
 	if (mm.is_valid() && dragging) {
+		if (!move_signal_sent) {
+			emit_signal("_begin_node_move");
+			move_signal_sent = true;
+		}
```

Comment:
Why are you introducing a new variable, if you have dragging available? Can't you check for `!dragging`?

Reply:
- `dragging` is always true in this block.
- Checking for `!dragging` in the `if (b->get_button_index() == BUTTON_LEFT && b->is_pressed()) {` if statement before `dragging` is set to true also doesn't work because that block is triggered by Ctrl+Clicking nodes
- `just_selected` and `!just_selected` also don't work because it can be either state depending on previous actions (e.g. if you ctrl+click a node, then drag it, `just_selected` will be false.  If you drag an unselected node, `just_selected` will be true)